core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt

```diff
@@ -139,15 +139,17 @@ fun LpOutlinedTextFieldMail(
  */
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun LpOutlinedTextFieldNumber(
+fun LpOutlinedTextFieldInput(
     modifier: Modifier,
+    enabled:Boolean,
```

Comment:
If you are going to modify a function that it is already been created, please add your name in the function documentation like this: `@modifyBy YourName`